slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/AbstractChatDeleteParams.java

```diff
@@ -7,18 +7,11 @@
 import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 import com.hubspot.immutables.style.HubSpotStyle;
-import com.hubspot.slack.client.methods.interceptor.HasChannel;
 
 @Immutable
 @HubSpotStyle
 @JsonNaming(SnakeCaseStrategy.class)
-public abstract class AbstractChatDeleteParams implements HasChannel {
-  @Override
-  @JsonProperty("channel")
-  public abstract String getChannelId();
-
-  @JsonProperty("ts")
-  public abstract String getMessageToDeleteTs();
+public abstract class AbstractChatDeleteParams extends AbstractChatDeleteParamsBase {
 
   @Default
   @JsonProperty("as_user")
```

Comment:
The current solution is 👍 but I wonder what do you think about annotating this property with `@JsonInclude(Include.NON_DEFAULT)`? This annotation says that if the value is the default we won't be serializing the property.

Reply:
Good point, didn't know about thit annotation.